### PR TITLE
Turku Old Events Importer

### PR DESCRIPTION
**Pull request by Anthon98**
Turku Old events importer implementation for importing old Turku Drupal events.
Removed old importers.
Documentation + code optimization will be added in the near future.

### CODE REVIEW

**Comment by Anthon98** [comment]
Documentation and better code optimization will be added in the near future.

File: events/importer/turku_old_events.py [hunk 1]
```diff
@@ -0,0 +1,794 @@
...
+
+                        if city and not addr.endswith(city):
+                            addr += '_%s' % city.lower()
+                        elif city and addr.endswith(city):
+                            ...
+                        elif not city and not addr.endswith('_turku'):
+                            addr += '_turku'
+
+                        origin_id = numeric(addr)
+                        tpr = '%s:%s' % (str(eventItem.get('data_source')), origin_id)  # Mimic tpr
```
Comment (ezkat) [inline events/importer/turku_old_events.py:489]: Is this still required?

File: events/importer/turku_old_events.py [hunk 2]
```diff
@@ -0,0 +1,794 @@
...
+                                event_keywords.add(Keyword.objects.get(id = ysoId[x]))
+                        else:
+                            event_keywords.add(Keyword.objects.get(id = ysoId))
+
+            if eventTku['keywords'] != None:
+                eventTku['keywords'] = eventTku['keywords'] + ','
+                keywords = eventTku['keywords'].split(',')
+                for name in keywords:
+                    if name[0:1] == " ":
+                        name = name.replace(name[0:1],"", 1)
```
Comment (ezkat) [inline events/importer/turku_old_events.py:395]: name.strip()

File: events/importer/turku_old_events.py [hunk 3]
```diff
@@ -0,0 +1,794 @@
...
+                    if name in TURKU_DRUPAL_CATEGORY_EN_YSOID.keys():
+                        ysoId = TURKU_DRUPAL_CATEGORY_EN_YSOID[name]
+                        if isinstance(ysoId, list):
+                            for x in range(len(ysoId)):
+                                event_keywords.add(Keyword.objects.get(id = ysoId[x]))
+                        else:
+                            event_keywords.add(Keyword.objects.get(id = ysoId))
+
+            if eventTku['keywords'] != None:
```
Comment (ezkat) [inline events/importer/turku_old_events.py:390]: if eventTku['keywords']:

File: events/importer/turku_old_events.py [hunk 4]
```diff
@@ -0,0 +1,794 @@
...
+
+            event_keywords = eventItem.get('keywords', set())
+            event_audience = eventItem.get('audience', set())
+
+            if eventTku['event_categories'] != None:
+                eventTku['event_categories'] = eventTku['event_categories'] + ','
+                categories = eventTku['event_categories'].split(',')
+                for name in categories:
+                    if name[0:1] == " ":
+                        name = name.replace(name[0:1],"", 1)
```
Comment (ezkat) [inline events/importer/turku_old_events.py:381]: name.strip()

File: events/importer/turku_old_events.py [hunk 5]
```diff
@@ -0,0 +1,794 @@
...
+                for k, v in x.items():
+                    if json_event['drupal_nid'] == str(k): #-> If event is a child.
+                        event_type = "child"
+                        #-> v is the childs mother
+                        for s in mothersUrl:
+                            for l, p in s.items():
+                                if v == l:
+                                    event_image_url = p
+
+            if event_type != None:
```
Comment (ezkat) [inline events/importer/turku_old_events.py:647]: if event_type:

File: events/importer/turku_old_events.py [hunk 6]
```diff
@@ -0,0 +1,794 @@
...
+                                        'info_url_en' : mother.info_url_fi,
+                                        'super_event' : mother}
+                                        )
+                                except Exception as ex: print(ex)
+                            except Exception as ex: pass
+
+                            try:
+                                # -> Get object from Event.
+                                child = Event.objects.get(origin_id=k)
+                                mother = Event.objects.get(origin_id=v)
```
Comment (ezkat) [inline events/importer/turku_old_events.py:699]: Unnecessary get, already fetched above.

Comment (Anthon98) [reply events/importer/turku_old_events.py:699]: I will keep the child get since child values got updated with update_or_create

**Pull request merged**